dev/tasks/tasks.yml

```diff
@@ -815,7 +815,10 @@ tasks:
     ci: github
     template: docker-tests/github.linux.yml
     params:
-      flags: -e ARROW_USE_MESON=ON
+      # ARROWCCACHE=OFF is for using sccache
+      flags: >-
+        -e ARROW_CCACHE=OFF
```

Comment:
But it's spelt `ARROW_USE_CCACHE` above. Are you sure this is working as intended?

Reply:
Thanks!

Reply:
I have confirmed in crossbow that this is now using sccache:

https://github.com/ursacomputing/crossbow/actions/runs/14175037282/job/39707754711#step:6:417